consensus/src/quorum_store/utils.rs

```diff
@@ -212,9 +209,9 @@ impl ProofQueue {
             {
                 // non-committed proof that is expired
                 num_expired_but_not_committed += 1;
-                if expiration_time.round() < current_time.round() {
-                    counters::GAP_BETWEEN_BATCH_EXPIRATION_AND_CURRENT_ROUND_WHEN_PULL_PROOFS
-                        .observe((current_time.round() - expiration_time.round()) as f64);
+                if expiration_time < current_block_timestamp {
```

Comment:
not related to this pr, but this function is ugly. it looks like we only need to do a single for loop over the digest_queue?